### ISSUE TITLE
Failed to find boxed file app/main.go

### ISSUE THREAD

**Issue by liuhengloveyou** [go-flutter-desktop/go-flutter#306]
## Hover doctor

```
Hover doctor
hover: Running on darwin
hover: Docker installed: false
hover: Sharing flutter version
Flutter 1.10.7 • channel beta • https://github.com/flutter/flutter.git
Framework • revision e70236e36c (5 周前) • 2019-10-02 09:32:30 -0700
Engine • revision 9e6314d348
Tools • Dart 2.6.0 (build 2.6.0-dev.0.0 1103600280)
hover: Flutter engine commit: https://github.com/flutter/engine/commit/9e6314d348
hover: Finding out the C compiler version
Apple clang version 11.0.0 (clang-1100.0.33.12)
Target: x86_64-apple-darwin19.0.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
hover: Sharing the content of go.mod
hover: Failed to read go.mod: open go/go.mod: no such file or directory
hover: Warning: No hover.yaml file found: open go/hover.yaml: no such file or directory
hover: Sharing the content of go/cmd
```

## Error output

hover init github.com/my-organization/simpleApplication, I get the following
error:

```log
hover: Failed to find boxed file app/main.go: open /go/pkg/mod/github.com/go-flutter-desktop/hover@v0.36.0/assets/app/main.go: no such file or directory
```

**Comment by pchampio** [comment]
Is golang newly installed?
Have you correctly set the $GOPATH? (and restarted the terminal..)

Duplicate of https://github.com/go-flutter-desktop/go-flutter/issues/257

**Comment by pchampio** [comment]
It might be because `GO111MODULE=on`.

**Comment by pchampio** [comment]
To reproduce this issue:
`GO111MODULE=on go get -u github.com/go-flutter-desktop/hover`

**Comment by pchampio** [comment]
might be related to: https://github.com/go-flutter-desktop/go-flutter/issues/257

**Comment by GeertJohan** [comment]
We could consider using `rice embed`. It writes the assets into .go files so that the files are embedded into the binary. It would require `rice embed` being run after each change to a template file, and the embed .go files must be committed to the repository for it to work with `go get`.

**Comment by jld3103** [comment]
That solutions seems good. The templates don't change that often and we can easily place a README into the templates folder

**Comment by pchampio** [comment]
We need a github action to create the exec!

**Comment by GeertJohan** [comment]
A github action on pr's xan also validate wether templates match embedded templates; otherwise error.

**Comment by GeertJohan** [comment]
This bug has been fixed by go-flutter-desktop/hover#101. After updating hover it should work. You can use `(cd $HOME && GO111MODULE=on go get -u -a github.com/go-flutter-desktop/hover)` to update to the new version of hover.
